README.md

```diff
@@ -2,30 +2,79 @@
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Write a loop to access each character in a string
+- Determine the presence of specific characters in a string
+- Test for characters that fall within a range of ASCII decimal values
 
 ## Introduction
 
-A 1-2 sentence summary of what will be covered.
+You will implement a simple password checker that measures
+the strength of a string based on how many of the following rules are met:
+
+1. The string length is at least 8.
+2. The string contains at least one special character '!' or '#' or '?'.
+3. The string contains at least one digit.
+4. The string contains at least one non-digit.
+
 
 ## Instructions
 
-Walk the student through any setup required to run the lesson (i.e.
-`npm install` and `npm start`).
+The starter project has a class `PasswordChecker` with several stub methods that return
+a default value.  You will adapt the methods to implement the password checker based on the
+rules above.   You are also provided a Junit test class `PasswordCheckerTest`, which
+has the test methods disabled.  As you solve each task, remove the `@Disabled` annotation
+to run the test method and confirm your implementation.
+
+Do not use regular expressions to solve any of the tasks.
```

Comment:
Just in case some students aren't familiar with regex, maybe add the note after this statement "We'll cover regular expressions in a later unit"? Limit confusion and lets them know if they don't know what it is, it's okay?

Reply:
ok